third-party/qthread/qthread-src/include/qthread/qthread.hpp

```diff
@@ -14,7 +14,11 @@
 template <bool> class OnlyTrue;
 template <> class OnlyTrue<true>
 {};
+#if __cplusplus >= 201103
```

Comment:
It does work under cce.
The nice thing about this test is that if it fails, it just falls back to the old version.